ui/components/multichain/create-account/create-account.js

```diff
@@ -43,7 +43,7 @@ export const CreateAccount = ({ onActionComplete }) => {
   const { isValidAccountName, errorMessage } = getAccountNameErrorMessage(
     accounts,
     { t },
-    newAccountName,
+    newAccountName === '' ? defaultAccountName : newAccountName,
```

Comment:
This could be shortened and clarified to:
`newAccountName ?? defaultAccountName`

Reply:
This ended up nullifying the change. The `??` operator only checks for null and undefined, but the `newAccountName` is always an empty string if it's unset (never nullish). So the default account name is never used here, leaving the behavior of the component exactly as it was before.

Reply:
Oh my!  Should I fix this?  I guess it should be `||` instead of `??`

Reply:
Yeah exactly! `||` should work well here. A fix would be appreciated.

We found this during RC testing but this improvement wasn't urgent, so we removed it from the RC. That is, fixing this isn't urgent, so no rush.